docs/guides/deployment.md

```diff
@@ -72,24 +87,58 @@ helm install --namespace vvp \
     --set vvpUrl=http://vvp-ververica-platform
 
 ## Community
+
+### For helm 2:
 helm install --namespace vvp \
     --name vp-k8s-operator \
     ./charts/vp-k8s-operator \
     --set vvpEdition=community \
     --set vvpUrl=http://vvp-ververica-platform
 
-# 5. 
-# Install the CRDs
-# Using the Cert created by the operator chart for serving webhooks.
-# Pointed at the webhook conversion service of the operator.
+### For helm 3:
+# Try dryrun to check the rendered templates
```

Comment:
Thanks for adding the Helm 3 docs! Do you think we could remove the dry run bits though? I think it would be best if people could just run each of these commands and have it deploy without modification.